internal/terraform/tfswitcher/install.go

```diff
@@ -15,7 +15,7 @@ import (
 const (
 	installFile               = "terraform"
 	versionPrefix             = "terraform_"
-	installPath               = ".terraform.versions"
+	installPath               = ".ize/.terraform.versions/"
```

Comment:
I'd say let's consider this:
```suggestion
	installPath               = ".ize/versions/terraform/"
```